src/test/java/com/saucelabs/bamboo/sod/action/BuildConfiguratorTest.java

```diff
@@ -29,10 +31,10 @@
 import java.util.List;
 import java.util.Map;
 
+import static com.saucelabs.bamboo.sod.config.SODKeys.TEMP_TUNNEL_ID;
 import static org.junit.Assert.*;
 import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
again the wildcard stuff - https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports